src/main/java/com/labs/iw/library/book/mutator/BookMutation.java

```diff
@@ -16,12 +20,24 @@
 	@Autowired
 	private BookRepository bookRepository;
 
-	public Book newBook(BookDTO bookDTO) {
+	@Autowired
+	private AuthorRepository authorRepository;
+
+	public Book newBook(BookDTO bookDTO,List<String> authorUuids) {
 		Book book = new Book();
 
 		book.setTitle(bookDTO.getTitle());
 		book.setDescription(bookDTO.getDescription());
 
+		List<Author> authors = new ArrayList<>();
```

Comment:
Please add a null or empty list check, if the authorUuids we should not try to add new author